build/dependencies.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <AspNetCoreVersion>$(VersionPrefix)-*</AspNetCoreVersion>
```

Comment:
I can swap the import order for dependencies.props and version.props in common.props, so long as this is actually an OK assumption to make.